### DISCUSSION TITLE
Automatically select Username, Password , No 2FA and Server

### DISCUSSION THREAD

**Discussion by RandAllTor** [Taxel/PlexTraktSync#1904] [Support] [answered]
Hello
Can someone tell me which file i should modify so i can put my Username, Password , No 2FA and Server so i can automate the process of running the script and make it as a cron job. Im running it on Ubuntu VM with Docker Compose.Now i have to run it manually and input these credentials every time i want to sync.

**Comment by glensc** [comment, marked as answer]
the files you need are in `Config Dir` in path shown by `plextraktsync info`.

also: this app does not store your password.

but rather seems typical https://xyproblem.info/

likely, your compose file doesn't setup `/app/config` as volume, therefore the config files don't persist across runs. you have not shared how you run nor your compose config.

**Comment by glensc** [comment]
user disappeared. marking answer and closing
